Source/WebKit/UIProcess/MiniBrowser.entitlements

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
What's this file?

Reply:
I think its like a configuration file created by webkit for mini browser